llama_index/extractors/interface.py

```diff
@@ -36,6 +37,11 @@ class BaseExtractor(TransformComponent):
         default=True, description="Whether to process nodes in place."
     )
 
+    num_workers: int = Field(
+        default=4,
```

Comment:
could we make it a variable to control

Reply:
It is actually -- it's exposed in each extractor since it extends this interface